scripts/deployment/arbitrum-local-template.json

```diff
@@ -17,7 +17,17 @@
                 "claim_request_extension": 3600,
                 "claim_period": 3600,
                 "challenge_period_extension": 3600
-            }
+            },
+            "token":
```

Comment:
Shouldn't this be called `"tokens"` (note the plural), since it's a list of tokens?